spec/models/hat_request_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe HatRequest do
+  it "has a limit on the hat field" do
+    hat_request = build(:hat_request, hat: "a" * 256)
+    expect(hat_request).to_not be_valid
```

Comment:
I just realized that I opened a PR that fixes a subset of your work.

I used the shoulda-matchers gem to get some cleaner specs written.

https://github.com/lobsters/lobsters/pull/750/files#diff-65783f9ae3d2c150b4912f3ebf364df4R5

Thoughts on switching to that gem for these tests?

Reply:
I would rather not use that for testing these validations.

I definitely think they read better but I feel like they don't show what the validation is actually doing as well since with the shoulda matchers you don't construct an invalid object.

Reply:
Ok, since you plan on doing more than me, I'll close out my PR.